experimental/OrthogonalDiscriminants/test/runtests.jl

```diff
@@ -4,3 +4,4 @@ using Test
 include("gram_det.jl")
 include("utils.jl")
 include("theoretical.jl")
+include("data.jl")
```

Comment:
CC @ThomasBreuer

Reply:
This is addressed by pull request #2800.

Reply:
PR #2800 now is merged.